core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -340,14 +340,21 @@ private static MediaType addKnownType(MediaType mediaType) {
      */
     public static final MediaType APPLICATION_XML_UTF_8 = createConstantUtf8(APPLICATION_TYPE, "xml");
 
+    public static final MediaType APPLICATION_XML = createConstant(APPLICATION_TYPE, "xml");
+
     public static final MediaType ATOM_UTF_8 = createConstantUtf8(APPLICATION_TYPE, "atom+xml");
+
+    public static final MediaType ATOM = createConstant(APPLICATION_TYPE, "atom");
+
     public static final MediaType BZIP2 = createConstant(APPLICATION_TYPE, "x-bzip2");
 
     /**
      * Media type for <a href="https://www.dartlang.org/articles/embedding-in-html/">dart files</a>.
      */
     public static final MediaType DART_UTF_8 = createConstantUtf8(APPLICATION_TYPE, "dart");
 
+    public static final MediaType DART = createConstant(APPLICATION_TYPE, "dart");
```

Comment:
I guess the constants you added here are probably not going to be used by users? If so, we could move them to `ArmeriaHttpUtil#HTTP_TEXTIBLE_MEDIA_TYPES`?